### ISSUE TITLE
Patreon support?

### ISSUE THREAD

**Issue by crimsonstrife** [mrbandler/TwitchAuth#20]
@mrbandler I realize this was aimed specifically at Twitch, but it got me thinking, would it be possible to do something similar with the Patreon API?

**Comment by mrbandler** [comment]
Hi @crimsonstrife, I just checked the Patreon API an authentication would certainly be possible, yes.
To obtain the complains a user is in would even be simpler than in the Twitch API.

I will at the feature later this month, maybe even this week depending on how smooth the transition of the current code base goes.

**Comment by mrbandler** [comment]
I will probably create a new repository named **SocialAuth** which will support Patreon and Twitch from the get-go. This is mostly due to the reason that I want to avoid duplicate code across multiple repos, which kind of have the same functionality.

As a lot of people already use TwitchAuth I will leave it as is and add a link to the new repository so they can update their plugin dependencies if they want to. This plugin will then be marked as deprecated.

@crimsonstrife  Any further Platform-Integration ideas are more than welcome!

**Comment by mrbandler** [comment]
**TL;DR**
New repo [here](https://github.com/mrbandler/PatreonAuth) and hit some problems but will continue to try to make the plugin a possibility.

**Long version...**
First of all, there is new repo with the Patreon code [here](https://github.com/mrbandler/PatreonAuth).
There is actually nothing there yet, as I was experimenting with it locally. I will push my changes to the dev branch as soon as I have a compilable state.

While experimenting I hit a major snag, the Patreon auth webpage does not like the embedded chromium browser which ships with UE4, as the reCAPTCHA widget is complaining about a "not supported" browser.
A workaround would be to open the webpage in a local browser on the player's system and let the player paste back the response to the game client, or try to render the page serverside... (I have to try that one).

Anyhow, I will continue to make the plugin however, it will not be finished this month, sorry.
